### PR TITLE
[Hot Fix] Increase Minimum Ethereum Withdraw Amount from 20 to 40 USDC

### PR DESCRIPTION
**Pull request by jp-mariano**
## Explanation of the solution
Based from an [announcement from Axelar](https://twitter.com/Axl_Satellite/status/1634510337557897218), we need to increase the minimum amount for Ethereum withdrawals.

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
- connect an admin wallet from `testnet` and verify that the schema check for Ethereum withdraw is now 40 USDC
- Binance withdraws should stay the same

## UI changes for review
Screenshot 2023-03-13 at 12 14 57

### CODE REVIEW

**Review by ap-justin — APPROVED** [review]

File: src/constants/contracts.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ const testnet:Contracts = {
   index_fund:      "juno1d26dmz9zld4n8pzjpqae5m95adnfalvz47vldpwfzsplw4umcjmqhkle94",
   cw4GrpApTeam:    "juno1qkphn8h2rnyqjjtfh8j8dtuqgh5cac57nq2286tsljducqp4lwfqac44jm",
   cw3ApTeam:       "juno1ukpw6g9dqfw3fk2mqchplgcz4jaukttf6tpyrht7sw00t7mn7dpsjwpfhs",
-  accounts:        "juno1cyd63pk2wuvjkqmhlvp9884z4h89rqtn8w8xgz9m28hjd2kzj2cqf070uz",
+  accounts:        "juno1d4v5h5yr920xjjwcuret5n3meee5l04c58fpjhghdgml8kp57spqvmg4lx",
```
Comment (SovereignAndrey) [inline src/constants/contracts.ts:30]: Contract address should not change for production.

**Review by SovereignAndrey — CHANGES_REQUESTED** [review]

File: src/constants/contracts.ts (hunk 1 above)
Comment (SovereignAndrey) [reply src/constants/contracts.ts:30]: Nm! I missed this was testnet configs. :facepalm:

Comment (SovereignAndrey) [inline src/constants/contracts.ts:30]: Don't need to change this on `master` for this PR. Would be better handled in separate PR that fixes all the contract addresses and not just Accounts. 
```suggestion
  accounts:        "juno1cyd63pk2wuvjkqmhlvp9884z4h89rqtn8w8xgz9m28hjd2kzj2cqf070uz",
```

**Review by SovereignAndrey — APPROVED** [review]

**Review by ap-justin — APPROVED** [review]
